### ISSUE TITLE
Oneplus 6T | Errors

### ISSUE THREAD

**Issue by anthonyaldazg** [Shubhamvis98/kali-pinephone#65]
Today I installed the latest version available in this repository. 

**ERRORS**:
1. USB port:
I connected a USB storage device via OTG but it is not recognized by the OS, neither is my external antenna **TP-LINK tl-wn-722n version 1.** `sudo dmesg -W` does not give any result when connecting a USB storage device or external antenna, only when connecting the USB charger.

2. Reboot:
Also sometimes when rebooting the OS the device displays '**Qualcomm CrashDump Mode**' on screen. To get out of it I had to reboot the device again.

3. Audio:
**Audio** does not work in 'firefox-esr', only in the default browser.

4. Update:
Unable to update packages with `sudo apt update`, it shows the following error:
![imagen](https://github.com/user-attachments/assets/f5d0ace3-7d34-499b-bfad-045a42b6a1fa)

**Comment by Shubhamvis98** [comment]
Try this release
https://github.com/Shubhamvis98/kali-pinephone/releases/tag/20241104

**Comment by odinhaknet** [comment]
Hi! I just tried the new version of NetHunter pro. By the way, the new version of the keyboard is awesome!
Fixed the audio problem in firefox-esr.

**Current errors:**
After rebooting a few times I get "Qualcomm CrashDump Mode" on the screen, maybe I just need to avoid rebooting a few times in a row.

I have downloaded the following packages and rebooted the OS, however, it still does not recognize my TP-LINK tl-wn722n version 1 antenna:
`sudo apt install bc build-essential libelf-dev linux-headers-$(uname -r) dkms firmware-atheros firmware-linux-free firmware-ath9k-htc -y `

Unable to update packages with sudo apt update, it shows the following error:
![imagen](https://github.com/user-attachments/assets/198bfbf5-16a4-4a33-8ab0-ff8cc824ca42)

**Comment by Shubhamvis98** [comment]
Those are not errors but warnings. You can fix it by removing non-free component from mobian.list repo file.
To avoid crash dump, try not to reboot instead you can use poweroff. I don't know actually what's the issue of crash dump.

And regarding your adapter, can you provide logs.

**Comment by odinhaknet** [comment]
It's like the USB port doesn't work. `dmesg -W` only records the charger cable

**Comment by Shubhamvis98** [comment]
have you checked the otg in android os? seems like there's some issue in your type-c port

**Comment by odinhaknet** [comment]
Hi, I just installed Android 10. I installed Magisk module for external network adapters. TP-LINK model tl-wn722n version 1 is recognized:
![Screenshot_20241111-101735](https://github.com/user-attachments/assets/01c3e815-6ad8-4640-a3b6-f8d7bca495f4)
![Screenshot_20241110-091846](https://github.com/user-attachments/assets/e8b5c5bd-aebb-4dc5-90c7-ea90f02e9385)

**Comment by odinhaknet** [comment]
I haven't tried another one. I only tried another one of the same model.

**Comment by Gyurma1988** [comment]
Hi, i have the same problem with Oneplus 6. No usb recognised. No wlan card no usb mass storage. I use the newest build.

**Comment by Shubhamvis98** [comment]
Anyone tried changing dr_mode in dts?

**Comment by beursdummy** [comment]
yes i have and it's work in progress : https://github.com/Shubhamvis98/kali-pinephone/issues/79
